### ISSUE TITLE
miner put hasrate to 0 

### ISSUE THREAD

**Issue by bilawalkazi** [develsoftware/GMinerRelease#397]
It seems like as soon as the miner discovers a card as LHR, it puts the card hash rate
![GPU2](https://user-images.githubusercontent.com/91029855/133940038-f119cff2-3da3-4766-b82c-3b2a32a9d152.png)
 to 0. I attached an image for review.

**Comment by george0329** [comment]
30秒後又會自動解鎖

**Comment by bilawalkazi** [comment]
yeah, it comes back to the original hashrate but you are losing hash rate for 30 seconds plus its happening to all cards, so every 30 seconds a card is 0% efficeint.

**Comment by george0329** [comment]
如果你一直觸發鎖定的話  要調整--lhr_tune1 2  我在--lhr_tune1 4或--lhr_tune1 5也都很容易觸發鎖定

**Comment by bilawalkazi** [comment]
Can you suggest any recommended OC settings for 3060 and 3060ti I have?
